qemu/tests/cfg/nvdimm.cfg

```diff
@@ -92,3 +93,13 @@
         - nvdimm_hotplug:
             del mem_devs
             target_mems = mem1
+        - nvdimm_nvml:
+            depends_pkgs = ndctl-devel daxctl-devel pandoc
+            nvml_test = yes
+            nvml_dir = /tmp/nvml
+            get_nvml = "git clone --depth=1 https://github.com/pmem/nvml.git ${nvml_dir}"
+            config_nvml = "cd ${nvml_dir}; make -j4"
+            config_nvml += ";cp src/test/testconfig.sh.example src/test/testconfig.sh"
+            config_nvml += ";echo 'PMEM_FS_DIR_FORCE_PMEM=1' >> src/test/testconfig.sh"
+            config_nvml += ";echo 'PMEM_FS_DIR=${mount_dir}' >> src/test/testconfig.sh"
+            run_test = "cd ${nvml_dir}; make check"
```

Comment:
Oh~   here is the `make check` step. 
The order should be `make`,  `make check`, `make install` (if needed), could you check the polarion case step ? Do we need `make install` ?

Reply:
We don't need make install. IIUC, Whether config before or after make is fine.

Reply:
I have no idea with the config and make, and the order of config and make.
This is about the `make install` too, same with the comment above.